enhancements/node-latency-profile/node-latency-profiles.md

```diff
@@ -0,0 +1,354 @@
+  // If the component enters in this state, "Default" WorkerLatencyProfileType
+  // rollout will be initiated to restore the respective default arguments of all
+  // components.
+  WorkerLatencyProfileDegraded NodeLatencyStatusConditionType = "Degraded"
+)
+```
+```go
+
+type ConditionStatus string
+
+// These are valid condition statuses. "ConditionTrue" means a resource is in the condition.
+// "ConditionFalse" means a resource is not in the condition. "ConditionUnknown" means kubernetes
+// can't decide if a resource is in the condition or not.
+const (
+	ConditionTrue    ConditionStatus = "True"
+	ConditionFalse   ConditionStatus = "False"
+	ConditionUnknown ConditionStatus = "Unknown"
+)
+```
+```go
+
+type NodeLatencyStatusCondition struct {
+	// type specifies the aspect reported by this condition.
+	// +kubebuilder:validation:Required
+	// +required
+	Type NodeLatencyStatusConditionType `json:"type"`
+
+	// status of the condition, one of True, False, Unknown.
+	// +kubebuilder:validation:Required
+	// +required
+	Status ConditionStatus `json:"status"`
+
+	// lastTransitionTime is the time of the last update to the current status property.
+	// +kubebuilder:validation:Required
+	// +required
+	LastTransitionTime metav1.Time `json:"lastTransitionTime"`
+
+	// reason is the CamelCase reason for the condition's current status.
+	// +optional
+	Reason string `json:"reason,omitempty"`
+
+	// message provides additional information about the current condition.
+	// This is only to be consumed by humans.  It may contain Line Feed
+	// characters (U+000A), which should be rendered as new lines.
+	// +optional
+	Message string `json:"message,omitempty"`
+}
+```
+```go
+// ObjectReference contains enough information to let you inspect or modify the referred object.
+type ObjectReference struct {
+	// group of the referent.
+	// +kubebuilder:validation:Required
+	// +required
+	Group string `json:"group"`
+	// resource of the referent.
+	// +kubebuilder:validation:Required
+	// +required
+	Resource string `json:"resource"`
+	// namespace of the referent.
+	// +optional
+	Namespace string `json:"namespace,omitempty"`
+	// name of the referent.
+	// +kubebuilder:validation:Required
+	// +required
+	Name string `json:"name"`
+}
+
+```
+#### WorkerLatencyProfile Custom Resource
+
+
+```yaml
+kind: WorkerLatencyProfile
+metadata:
+  name: medium-update-and-average-reaction
+spec:
+  profileType: "MediumUpdateAndAverageReaction" # Possible values, "MediumUpdateAndAverageReaction", "LowUpdateAndSlowReaction" or "Default"
+```
+
+### Operational Aspects of API Extensions
+
+Openshift always assumes low latency with 100% reliable network connections amongst master nodes. However, this may not be always true for the worker nodes (e.g Edge cases). In this section we will discuss how to keep the cluster healthy when the worker nodes have relatively high latency with not so reliable network connections to reach the master nodes.
```

Comment:
this is a good discussiion (please keep it), but it is not really what belongs into this section. The section is about what to do when a cluster with a custom profile is running into problems. How can one rule out that this feature is the cause of the problem? Can one go back to the normal profile and try without dataloss and without outage?